### DISCUSSION TITLE
scanf funksiyonu

### DISCUSSION THREAD

**Discussion by B3Y0N4** [19/classroom#59] [C] [answered]
scanf ("%d ", ptr + 1); ile scanf ("%d", ptr + 1); neden farkli çalişir
 "%d " bır boşluk eklediğmizde nasil çalişiyor scanf

**Comment by roktas** [comment, marked as answer]
`scanf`'in davranışındaki uç durumları öğrenmek için kılavuz sayfasını okumalısın: https://linux.die.net/man/3/scanf

İlgili bölüm:
![image](https://user-images.githubusercontent.com/47842/197401677-891ffa8a-4643-437f-9978-73471934471d.png)

**Reply by B3Y0N4** [reply]
tamamdir teşşekür ederim
